x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/CcrRepositoryIT.java

```diff
@@ -235,6 +238,60 @@ public void testDocsAreRecovered() throws Exception {
         thread.join();
     }
 
+    public void testRateLimitingIsEmployed() throws Exception {
+        restartClustersWithSettings(Settings.builder().put(CcrSettings.FOLLOWER_RECOVERY_MAX_BYTES_READ_PER_SECOND.getKey(),
```

Comment:
by making the setting truly dynamic, we won't need the restarts here, which will greatly speed up the tests. See SharedClusterSnapshotRestoreIT#testThrottling for how it's tested for snapshot /restore.

Reply:
I did the same setup and assertion as that test.